contributors/devel/e2e-tests.md

```diff
@@ -115,7 +114,7 @@ GINKGO_PARALLEL=y go run hack/e2e.go -- -v --test --test_args="--ginkgo.skip=\[S
 # You can also specify an alternative provider, such as 'aws'
 #
 # e.g.:
-KUBERNETES_PROVIDER=aws go run hack/e2e.go -- -v --build --up --test --down
+go run hack/e2e.go -- -v --build --up --test --down --provider=aws
```

Comment:
Yeah, that works fine. I confirmed it like:

$ go run hack/e2e.go -- --provider=skeleton -v --test  --test_args="--ginkgo.focus=blank"
2017/08/17 09:35:03 e2e.go:76: Calling kubetest --provider=skeleton -v --test --test_args=--ginkgo.focus=blank...
2017/08/17 09:35:03 util.go:131: Running: ./cluster/kubectl.sh --match-server-version=false version
2017/08/17 09:35:03 util.go:133: Step './cluster/kubectl.sh --match-server-version=false version' finished in 76.603564ms
2017/08/17 09:35:03 util.go:131: Running: ./hack/e2e-internal/e2e-status.sh
[..]
Ran 1 of 675 Specs in 11.386 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 674 Skipped PASS

Ginkgo ran 1 suite in 11.583277352s
Test Suite Passed
2017/08/17 09:35:15 util.go:133: Step './hack/ginkgo-e2e.sh --ginkgo.focus=blank' finished in 11.813172925s
2017/08/17 09:35:15 e2e.go:80: Done
$

I will move the option as you pointed.